docs/next/modules/en/pages/setup/install-stackstate/kubernetes_openshift/affinity.adoc

```diff
@@ -0,0 +1,128 @@
+
+[,yaml]
+----
+affinity:
+  podAntiAffinity:
+    # Enable required pod anti-affinity (true = hard, false = soft)
+    requiredDuringSchedulingIgnoredDuringExecution: true
+    # Topology key for pod anti-affinity
+    topologyKey: "kubernetes.io/hostname"
+----
+
+== Behavior
+
+=== Node Affinity
+* **When configured**: Applied to all components
+
+=== Pod Anti-Affinity
+* **When configured**: Only applied when `sizing.profile` ends with `-ha` (High Availability profiles)
+* **HA Profiles**: `150-ha`, `250-ha`, `500-ha`, `4000-ha`
+* **Components affected**: All stateful data services including Clickhouse, Kafka, Zookeeper, VictoriaMetrics, StackGraph, Elasticsearch
+
+== Example Configurations
+
+=== Basic Node Affinity (Same Availability Zone) + Hard PodAntiAffinity for HA Deployment
+Assuming `sizing.profile` is set to one of the HA profiles, the values below will configure nodeAffinity to schedule components of SUSE Observability to `us-west-2a` zone and enforce scheduling the replicas of the same data services to different hosts (`affinity.podAntiAffinity.requiredDuringSchedulingIgnoredDuringExecution`=`true` by default).
+[,yaml]
+----
+affinity:
+  # Schedule all pods to nodes in the same AZ
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: topology.kubernetes.io/zone
+          operator: In
+          values:
+          - us-west-2a
+----
+
+=== Soft PodAntiAffinity for HA Deployment
+If for any reason enforcing scheduling of data services to different nodes is not desirable, podAntiAffinity can be configured in "soft" mode:
+[,yaml]
+----
+affinity:
+  podAntiAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution: false
+----
+
+== Usage
+
+=== Step 1: Create Your Affinity Values File
+
+Create a separate values file with your desired affinity configuration. For example, save the following as `suse-observability-values-values.yaml`:
+
+[,yaml]
+----
+affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: topology.kubernetes.io/zone
+          operator: In
+          values:
+          - us-west-2a
+  podAntiAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution: true
+----
+
+=== Step 2: Generate Affinity Template Values
+
+Run the following command to generate the affinity values template:
+
+[,text]
+----
+export VALUES_DIR=.
+helm template \
+  --set license='<your license>' \
+  --set baseUrl='<suse-observability-base-url>' \
+  --set sizing.profile='<sizing.profile>' \
+  --values suse-observability-values-values.yaml \
+  suse-observability-values \
+  suse-observability/suse-observability-values --output-dir $VALUES_DIR
+----
+
+=== Step 3: Use Generated Values in Helm Installation
```

Comment:
The idea was that the release note for anti-affinity links to this heading. This should be possible by just using a relative link to this file.